tests/test_assessment.py

```diff
@@ -9030,6 +9030,60 @@ def test_can_submit_foreign_language_text_response(self):
         self.assertTrue(data['correct'])
         self.assertIn('<p/>', data['feedback'])
 
+    def test_can_edit_max_strings(self):
+        item = self.create_item()
+        old_max_strings = item['question']['maxStrings']
```

Comment:
It's a nit really.  I was thinking that the test doesn't control its inputs so if the default value was to change to 55 (unlikely) then the test would be unreliable.